### PR TITLE
Yieldmo: Add support for gpp and gpp_sid

### PR DESCRIPTION
**Pull request by ym-prasanth**

### CODE REVIEW

File: static/bidder-info/yieldmo.yaml [hunk 1]
```diff
@@ -13,5 +13,5 @@ capabilities:
     - video
 userSync:
   redirect:
-    url: "https://ads.yieldmo.com/pbsync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirectUri={{.RedirectURL}}"
+    url: "https://ads.yieldmo.com/pbsync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirectUri={{.RedirectURL}}"
```
Comment (gargcreation1992) [inline static/bidder-info/yieldmo.yaml:16]: user sync url doesn't work for me. It says 400 bad request - 
Screenshot 2024-02-05 at 4 31 34 PM

Here is an URL that I am trying -
`https://[ads.yieldmo.com](https://ads.yieldmo.com/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dyieldmo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID)/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dyieldmo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID`

Comment (ym-prasanth) [reply static/bidder-info/yieldmo.yaml:16]: Your redirect uri is messed up. Try this: https://ads.yieldmo.com/pbsync?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirectUri=https%3A%2F%2Fwww.cnn.com%3Fuid%3D%24UID

**Review by gargcreation1992 — APPROVED** [review]

File: static/bidder-info/yieldmo.yaml (hunk 1 above)
Comment (ym-prasanth) [reply static/bidder-info/yieldmo.yaml:16]: @gargcreation1992 Can you please merge this?

Comment (ym-prasanth) [reply static/bidder-info/yieldmo.yaml:16]: @gargcreation1992 @onkarvhanumante Any idea when this will be merged? It's blocking us. Thanks.

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**